.scripts/check-ts-changes.sh

```
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Get list of staged and unstaged TS/TSX files
+CHANGED_TS_FILES=$(git diff --name-only --diff-filter=ACMR | grep -E \"\\.tsx?$\" || true)
```

Shellcheck (0.10.0)

[warning] Quote the grep pattern so the shell won't interpret it.

(SC2062)